### DISCUSSION TITLE
brandymedia/turbine-ui-core

### DISCUSSION THREAD

**Discussion by brandymedia** [HassanZahirnia/laravel-package-ocean#157] [Package Suggestions]
Hey there, I'd like to suggest a new Laravel component UI library called Turbine UI.

Links:
Homepage: turbineui.com
GitHub: https://github.com/brandymedia/turbine-ui-core

Please let me know if you need any further details.

**Comment by HassanZahirnia** [comment]
Hey, can you tell me why there is no readme or further information in the repo? Or any website?

**Comment by brandymedia** [comment]
Hi Hassan,

The website URL was in the first post turbineui.com

I’ll update the repo to add a readme… was on my todo list.

Until then, there is the docs -
https://turbineui.com/documentation

If you need anything else, please let me know 👍

**Reply by HassanZahirnia** [reply]
Thanks for the information. I've already added your package, but I think you'll have to add some information to the repo. Because when users click on the link, there has to be some link to the website and some minimal readme information.

Happy coding!

**Comment by brandymedia** [comment]
Great thanks Hassan.

I see what you mean. I’ll get that updated asap.

**Reply by HassanZahirnia** [reply]
Thanks for adding info to the repo!

**Reply by brandymedia** [reply]
No problem. Thanks for including on Laravel Package Ocean 💙
